File: KennyWambua/fizz_buzz.py
"""

    A python program that prints the numbers from 1 to 100. For multiples of 3, print Fizz;
    For multiples of 5, print Buzz; and for numbers that are multiples of 3 and 5 , print FizzBuzz.

"""

def fizz_buzz():
    for num in range(1, 101):
        if num % 3 == 0 and num % 5 == 0:
            print("FizzBuzz")
        elif num % 3 == 0:
           print("Fizz")
        elif num % 5 == 0:
            print("Buzz")
        else:
            print(num)

if __name__ == "__main__":
    fizz_buzz()

File: CollinsOmwoyo/fizz_buzz.c
#include <stdio.h>

/**
 * main - Prints numbers from 1 to 100 with special cases for multiples of 3 and 5.
 *
 * Return: Always 0 (Success)
 */
int main(void)
{
    for (int i = 1; i <= 100; i++)
    {
        if (i % 3 == 0 && i % 5 == 0)
            printf("FizzBuzz");
        else if (i % 3 == 0)
            printf("Fizz");
        else if (i % 5 == 0)
            printf("Buzz");
        else
            printf("%d", i);

        if (i < 100)
            printf(" ");  // Print space between outputs except after the last one.
    }
    printf("\n");
    return 0;
}
